po/common/common.pot

```diff
@@ -3284,6 +3284,34 @@ msgctxt "get_the_app_ipad"
 msgid "Get the iPad app"
 msgstr ""
 
+msgctxt "producer_product_id"
```

Comment:
Unused translation L3287-3313?

Reply:
Those are translations I had forgotten to add before (for the import of products from Carrefour)